packages/@o3r/testing/schematics/ng-add/fixture/index.ts

```diff
@@ -49,23 +46,60 @@ export function updateFixtureConfig(options: { projectName?: string | null | und
         return libs;
       }, []);
 
-      tsconfigCompilerOptions.paths = Object.entries(tsconfigCompilerOptions.paths || {}).reduce<Record<string, unknown>>((acc, [key, value]) => {
-        if (oldPaths.indexOf(key) === -1) {
-          acc[key] = value;
+      const testFramework = options.testingFramework || getTestFramework(getWorkspaceConfig(tree), context);
+      if (testFramework === 'jest') {
+        tsconfigCompilerOptions.types ||= [];
```

Comment:
this may break the build in some cases
> By default all visible ”@types” packages are included in your compilation. Packages in node_modules/@types of any enclosing folder are considered visible. For example, that means packages within ./node_modules/@types/, ../node_modules/@types/, ../../node_modules/@types/, and so on.
If types is specified, only packages listed will be included in the global scope.